### PR TITLE
Implementation of some editor.action commands needed for plugin API

### PR DESCRIPTION
**Pull request by JPinkney**
Signed-off-by: Josh Pinkney <joshpinkney@gmail.com>

This PR makes 'editor.action.rename', 'editor.action.formatSelection', 'editor.action.formatDocument' available.

Related issue: https://github.com/eclipse/che/issues/13007, https://github.com/theia-ide/theia/issues/5215, https://github.com/theia-ide/theia/issues/5216

### CODE REVIEW

File: packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts [hunk 1]
```diff
@@ -309,6 +309,22 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
          * Show Opened File in New Window	workbench.action.files.showOpenedFileInNewWindow
          * Compare Opened File With	workbench.files.action.compareFileWith
          */
+
+        commands.registerCommand({ id: 'editor.action.rename' }, {
```
Comment (benoitf) [inline packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts]: hello, are these commands should be handled more globally https://github.com/theia-ide/theia/blob/33590a9a8fc9899fdd0f0ed57cb3ea4871adf6e1/packages/monaco/src/browser/monaco-command.ts ?

Comment (JPinkney) [reply packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts]: @benoitf Fixed. I wasn't exactly sure where to define the commands so I defined them with the editor commands. If they should be somewhere else let me know!

**Comment by akosyakov** [comment]
We have a PR to expose monaco editor commands: https://github.com/theia-ide/theia/pull/4275/files#diff-50091b54c279f7fa9bf0acce18541c06R243 But decided against doing it in the editor extension: https://github.com/theia-ide/theia/pull/4275#issuecomment-461413431

**Comment by JPinkney** [comment]
@akosyakov That looks like a much better solution and gives us a lot more commands. For keybindings coming from VSCode keybinding plugins they all use something like 'editor.action.formatSelection' [1] rather than 'monaco.editor.action.formatSelection' which means almost all of the keybindings won't register.

[1] - https://github.com/alphabotsec/vscode-eclipse-keybindings/blob/master/package.json#L178

**Pull request closed without merging**